### PR TITLE
[docs] Fix missing interpreter_main flags in hello xls tutorial.

### PR DESCRIPTION
**Pull request by schilkp**
Small addition to the "Hello XLS" tutorial: The `trace!` output is not visible by default, and requires `--alsologtostderr` to be seen.

### CODE REVIEW

File: docs_src/tutorials/hello_xls.md [hunk 1]
```diff
@@ -119,6 +119,9 @@ trace of hello_string @ hello.x:4:17-4:31: [72, 101, 108, 108, 111, 44, 32, 88,
 [===============] 1 test(s) ran; 0 failed; 0 skipped.
 ```
 
+Note that the `trace!` output is treated as `INFO` level logging,
+which is printed to stderr by adding the '--alsologtostderr', making it visible.
```
Comment (mikex-oss) [inline docs_src/tutorials/hello_xls.md:123]: Nit: the multiple clauses is a bit wordy. 

How about "which is made visible in stderr by adding the '--alsologtostderr'.

Comment (schilkp) [reply docs_src/tutorials/hello_xls.md]: Sure! Fixed + force-pushed :)
